packages/apollo-language-server/src/config/__tests__/loadConfig.ts

```diff
@@ -0,0 +1,239 @@
+import { loadConfig } from "../";
+import * as fs from "fs";
+
+const makeNestedDir = dir => {
```

Comment:
Since they're in separate `packages`, I'm not sure. Do we have a pattern for doing that anywhere without putting it in a util package like `apollo-env`? I'm not sure how we feel about putting things at the root of a monorepo.

Reply:
You're right - that's an oversight on my part. LGTM.